src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBaseWithWiremock.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.darts.testutils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import uk.gov.hmcts.darts.testutils.stubs.wiremock.DartsGatewayStub;
+
+// port = 0 enables random ports as recommended by Wiremock. Tests will be faster and more reliable
+@AutoConfigureWireMock(port = 0, files = "file:src/integrationTest/resources/wiremock")
+public class IntegrationBaseWithWiremock extends IntegrationBase {
```

Comment:
author note: 
I've renamed `IntegrationBaseWithGatewayStub` to `IntegrationBaseWithWiremock` and designated it as the base test class for tests requiring Wiremock